content/en/platform/corda/5.1/application-networks/creating/members/reregister.md

```diff
@@ -0,0 +1,244 @@
+request was approved by checking the status of the registration request.
+
+After successful re-registration, you should be able to see the member's information containing the `ext.sample` field in
+their member-provided context. The member retains its most recent status in the group, for example, a suspended member
+will remain suspended after successful re-registration.
+
+## **Optional:** Include Serial Number in the Registration Context
+
+The `corda.serial` Corda platform property is embedded in the `MemberInfo` of all members. It acts as the `MemberInfo`'s version,
+and is incremented by 1 after each registration.
+
+This serial number may be optionally included in the registration context to specify which version of the `MemberInfo` was
+intended to be updated. If no serial number is provided while re-registering, the platform will use the member's latest
+serial number by default. Only the latest `MemberInfo` version may be updated - requests with an older serial number are declined.
+It is recommended to provide the serial number to avoid unintentional updates, in case you have an outdated version of the `MemberInfo`.
+
+You can retrieve the serial number from the MGM-provided context of the `MemberInfo` by performing a member lookup.
+
+{{< note >}}
+For first-time registration the serial number should be 0, if provided.
+{{< /note >}}
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```bash
+export REGISTRATION_CONTEXT='{
+  "corda.session.keys.0.id": "'$SESSION_KEY_ID'",
+  "corda.session.keys.0.signature.spec": "SHA256withECDSA",
+  "corda.ledger.keys.0.id": "'$LEDGER_KEY_ID'",
+  "corda.ledger.keys.0.signature.spec": "SHA256withECDSA",
+  "corda.endpoints.0.connectionURL": "https://'$P2P_GATEWAY_HOST':'$P2P_GATEWAY_PORT'",
+  "corda.endpoints.0.protocolVersion": "1",
+  "ext.sample": "apple",
+  "corda.serial": "1"
+}'
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+$REGISTRATION_CONTEXT = @{
+  'corda.session.keys.0.id' =  $SESSION_KEY_ID
+  'corda.session.keys.0.signature.spec' = "SHA256withECDSA"
+  'corda.ledger.keys.0.id' = $LEDGER_KEY_ID
+  'corda.ledger.keys.0.signature.spec' = "SHA256withECDSA"
+  'corda.endpoints.0.connectionURL' = "https://$P2P_GATEWAY_HOST`:$P2P_GATEWAY_PORT"
+  'corda.endpoints.0.protocolVersion' = "1",
+  'ext.sample' = "apple",
+  'corda.serial' = "1"
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+## Request Queue
+
+If a member submits more than one registration request at the same time, the MGM will queue the requests and process them
+one by one, treating each subsequent request in the queue as a re-registration attempt.
+
+If a member submits multiple re-registration requests with the same serial number, the first request will be processed
+(if the serial number is valid). However, the other requests will be declined because the serial number specified in
+those requests would be outdated after the first request is completed.
+
+## MGM Admin API
```

Comment:
This section should probably be under Managing Members, as this API is for the MGM operator

Reply:
Moved